### ISSUE TITLE
Support for Angular 8?

### ISSUE THREAD

**Issue by MSafter** [angular-redux/store#562]
### This is a...

* [ ] feature request
* [ ] bug report
* [x] usage question

### What toolchain are you using for transpilation/bundling?

* [x] @angular/cli
* [ ] Custom @ngTools/webpack
* [ ] Raw `ngc`
* [ ] SystemJS
* [ ] Rollup
* [ ] Other

### Environment

NodeJS Version: 12.3.1
Typescript Version: 3.4.5
Angular Version: 8.0.0
@angular-redux/store version: 10.0.0
@angular/cli version: (if applicable)
OS: macOS 

### Link to repo showing the issus
(optional, but helps _a lot_)

### Expected Behaviour:
@angular-redux/store is working fine with Angular 8

### Actual Behaviour:
Seems not to work with Angular 8

### Stack Trace/Error Message:
zone-evergreen.js:359 Uncaught (in promise) TypeError: Class constructor Observable cannot be invoked without 'new'
    at new ActionsObservable (ActionsObservable.js:30)
    at epicMiddleware (createEpicMiddleware.js:31)
    at redux.js:615
    at Array.map (<anonymous>)
    at redux.js:614
    at RootStore.configureStore (angular-redux-store.js:584)
    at new StoreModule (store.module.ts:158)
    at _createClass (core.js:23183)
    at _createProviderInstance (core.js:23137)
    at initNgModule (core.js:23043)
    at new NgModuleRef_ (core.js:24175)
    at createNgModuleRef (core.js:24158)
    at Object.debugCreateNgModuleRef [as createNgModuleRef] (core.js:35695)
    at NgModuleFactory_.create (core.js:36820)
    at core.js:31585
    at ZoneDelegate.invoke (zone-evergreen.js:359)
    at Object.onInvoke (core.js:30892)
    at ZoneDelegate.invoke (zone-evergreen.js:358)
    at Zone.run (zone-evergreen.js:124)
    at NgZone.run (core.js:30757)
    at PlatformRef.bootstrapModuleFactory (core.js:31578)
    at core.js:31648
    at ZoneDelegate.invoke (zone-evergreen.js:359)
    at Zone.run (zone-evergreen.js:124)
    at zone-evergreen.js:855
    at ZoneDelegate.invokeTask (zone-evergreen.js:391)
    at Zone.runTask (zone-evergreen.js:168)
    at drainMicroTaskQueue (zone-evergreen.js:559)
    at ZoneTask.invokeTask [as invoke] (zone-evergreen.js:469)
    at invokeTask (zone-evergreen.js:1603)
    at HTMLScriptElement.globalZoneAwareCallback (zone-evergreen.js:1629)

### Additional Notes:
(optional)

**Comment by alaindeurveilher** [comment]
Shouldn't this issue be reported in https://github.com/angular-redux/platform instead?

**Comment by MSafter** [comment]
@AlainD- yeah you're right. I've hit the new Issue in the wrong repo. Sorry my bad :/ Can we move the issue?

**Comment by JaydeSilvaEmpire** [comment]
I am facing this issue too, any progress on this?

**Comment by joschne** [comment]
I was able to solve this issue with [redux-observable-es6-compat](https://www.npmjs.com/package/redux-observable-es6-compat)

**Comment by maplion** [comment]
@joschne I did the same.
